docs/RELEASE.md

```diff
@@ -0,0 +1,77 @@
+# Release
+
+This document aims to describe how to perform a release of the Horreum python library.
+
+The versioning pattern should follow the [Horreum](https://github.com/Hyperfoil/Horreum) 
+versioning scheme to keep coherence among all Horreum-related projects versions.
+
+## Procedure
+
+### Tag a new version
+
+Checkout to your branch, either a "stable" (e.g., `0.12.x`) or the `main` one.
+
+```bash
+git checkout origin/main
+```
```

Comment:
You really should, first, do a command like
```
git remote update [--prune]
```
to ensure that the `origin/main` tracking branch is actually up to date with the remote `origin` repo before you create a local branch from it.

Then, when you do the checkout, you need to specify the name of the local branch to create (otherwise, at least in my environment, you end up in "detached HEAD state"):
```
git checkout origin/main -b main
```
(If you maintain a local `main` branch which you keep up to date with the `origin/main` by doing periodic `pull`'s, perhaps this doesn't apply -- I don't keep a `main` branch, locally -- but otherwise I think you'll run into problems when you use a branch other than `main` if you don't already have it and maintain it locally.)